examples/iDynTreeWrappers/visualizeRobot.m

```diff
@@ -0,0 +1,111 @@
+%% This script is an example on how to use the matlab iDynTree visualization
+%%      Launch this file from the folder it is contained
+currentFolder=pwd;
+% insert the path where the meshes may be found, it replaces what is required
+% to complete the path inside the urdf visual mesh field.
+
+% Example: 
+% The test model has the following mesh file in the URDF
+% 'package://stl/sim_sea_2-5_root_link_prt-binary.stl'
+% The required file is inside the meshes folder in this directory,
+% so to complete the path we are missing meshes as a prefix to the full
+% path starting from this directory.
+meshFilePrefix='meshes/';
+% The model is in this same folder
+modelPath=[currentFolder '/'];
+fileName='model_uses_stl.urdf';
```

Comment:
Sorry, these meshes are too heavy to be loaded in the iDynTree repo. Can't we just keep the code that is in the  later comment?

Reply:
Ok